tests/README.md

```diff
@@ -17,12 +17,12 @@ In order to run the tests you need to provide the binaries for both the plugin a
 
 Compiling the plugin will produce the binary for the targeted device. They will be used as-is by the test framework.
 
-The binaries of the Ethereum application must be gathered and placed in the directory `tests/ethereum_build/`.
+The binaries of the Ethereum application must be gathered and placed in the directory `tests/.test_dependencies/`.
 Example of the correct file tree with Ethereum compiled for all targets.
-* `tests/ethereum_build/build/nanos/bin/app.elf`
-* `tests/ethereum_build/build/nanos2/bin/app.elf`
-* `tests/ethereum_build/build/nanox/bin/app.elf`
-* `tests/ethereum_build/build/stax/bin/app.elf`
+* `tests/.test_dependencies/build/nanos/bin/app.elf`
```

Comment:
This is not correct, the tree is not `build/<device>/bin/app.elf` but rather `<repo-name><use-case>/build/<device>/bin/app.elf`